--- splashutils-1.5.4.4-r4.ebuild	2014-04-02 20:30:34.185227734 +0100
+++ splashutils-1.5.4.4-r5.ebuild	2014-04-02 20:32:11.979196388 +0100
@@ -90,6 +90,13 @@
 		epatch "${FILESDIR}/splashutils-1.5.4.4-freetype-bz2.patch"
 	fi
 
+	if has_version ">=media-libs/freetype-2.5.1" ; then
+		cd "${SM}"
+		epatch "${FILESDIR}/splashutils-1.5.4.4-freetype251.patch"
+		cd "${S}"
+		epatch "${FILESDIR}/splashutils-1.5.4.4-freetype251-splashutils.patch"
+	fi
+
 	cd "${S}"
 	ln -sf "${S}/src" "${WORKDIR}/core"
 
